src/uct/ib/base/ib_iface.c

```diff
@@ -1641,11 +1642,15 @@ uct_ib_iface_estimate_perf(uct_iface_h iface, uct_perf_attr_t *perf_attr)
         break;
     default:
         send_pre_overhead        = iface_attr.overhead;
-        send_post_overhead_bcopy = send_post_overhead_zcopy = 0;
+        send_post_overhead_bcopy = 40e-9;
```

Comment:
is it only for calling packing function and obtaining bounce buffer?

Reply:
it's for the cost of doorbell, actually "bcopy" also includes short case (see lines 1657-1661)

Reply:
maybe worth adding a comment that this is for doorbell?
also what does UCT report as a software overhead in iface_attr then?

Reply:
fixed (force push)
iface_attr.overhead is the preparation of posting the doorbell, and post_overhead would be the cost of writing to IO memory and its effect on instruction pipeline